ent/schema/balance.go

```diff
@@ -0,0 +1,37 @@
+package schema
+
+import (
+	"entgo.io/contrib/entgql"
+	"entgo.io/ent"
+	"entgo.io/ent/schema"
+	"entgo.io/ent/schema/edge"
+	"entgo.io/ent/schema/field"
+)
+
+type Balance struct {
```

Comment:
this will make this sort of schema a bit nicer: https://github.com/ent/ent/pull/2560